tests/codegen/try_question_mark_nop.rs

```diff
@@ -7,11 +7,14 @@
 use std::ops::ControlFlow::{self, Break, Continue};
 use std::ptr::NonNull;
 
+// FIXME: The `trunc` and `select` instructions can be eliminated.
 // CHECK-LABEL: @option_nop_match_32
 #[no_mangle]
 pub fn option_nop_match_32(x: Option<u32>) -> Option<u32> {
     // CHECK: start:
-    // CHECK-NEXT: insertvalue { i32, i32 }
+    // CHECK-NEXT: [[TRUNC:%.*]] = trunc nuw i32 %0 to i1
+    // CHECK-NEXT: [[FIRST:%.*]] = select i1 [[TRUNC]], i32 %0
```

Comment:
Fixed by https://github.com/llvm/llvm-project/pull/105914 (LLVM 20).

Reply:
Fwiw I have been using e.g. `FIXME(llvm20)` for things that should be adjusted when we upgrade. But I don’t think I have seen anybody else do this and I still forget to update them, so grain of salt 😀

Reply:
Ah, I just noticed this is a regression from the job `x86_64-gnu-llvm-17`. I'll still try to resolve it in the MIR.